<ui version="4.0" >
 <class>ErrorDialog</class>
 <widget class="QDialog" name="ErrorDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Error - NanoVision-1</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="errorImageLabel" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap>Icons/error.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel" >
       <property name="text" >
        <string>errorLabel</string>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Details:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ErrorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ErrorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
